### PR TITLE
Add a new blog post morning star

### PR DESCRIPTION
**Pull request by codewithkushagra**
### All Submissions:

* [ ] Have you followed the guidelines in our Contributing document?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### New Feature Submissions:

1. [ ] Does your submission pass tests?
2. [ ] Have you lint your code locally prior to submission?

### Changes to Core Features:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -81,6 +83,14 @@ authors:
...
     twitter:        https://www.instagram.com/kushagra_agra008/
 
+  kushagra:
+    name:           Kushagra Agarwal
+    site:           https://codewithkushagra.netlify.app/
+    avatar:         "assets/images/authors/kushagra.jpg"
+    bio:            "I am a code geek, a NARUTO fan and also love expressing my depression in words :)"
+    email:          kushagraagra008@gmail.com
+    twitter:        https://www.instagram.com/kushagra_agra008/
+
```
Comment (AvikantSrivastava) [inline _config.yml]: Why have you added your info again?

File: _config.yml [hunk 2]
```diff
@@ -43,6 +44,7 @@ authors:
     email:          adarshsingh22032002@gmail.com
     twitter:        https://www.instagram.com/just_adarsh22/
 
+
```
Comment (AvikantSrivastava) [inline _config.yml]: remove this line

File: _config.yml [hunk 3]
```diff
@@ -35,6 +35,7 @@ authors:
     email:          adarshkgupta@outlook.com
     twitter:        https://www.instagram.com/adkgup/
 
+
```
Comment (AvikantSrivastava) [inline _config.yml]: remove this line

File: Gemfile.lock [hunk 4]
```diff
@@ -0,0 +1 @@
+
```
Comment (AvikantSrivastava) [inline Gemfile.lock]: remove this line

**Review by AvikantSrivastava — CHANGES_REQUESTED** [review]

File: _config.yml (hunk 1 above)
Comment (codewithkushagra) [reply _config.yml]: Oh, okay I didn't realised I will make the correction

File: Gemfile.lock (hunk 4 above)
Comment (codewithkushagra) [reply Gemfile.lock]: made all the changes please review my PR @AvikantSrivastava  bhaiya

**Review by AvikantSrivastava — APPROVED** [review]
Amazing, changes look perfect

**Pull request merged**